code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -506,10 +506,19 @@
 	flags_inv = HIDEHAIR|HIDEEARS
 
 /obj/item/clothing/suit/hooded/wintercoat/centcom
```

Comment:
The changes done to this subtype are too excessive. This is not meant for deathsquads.